### PR TITLE
Feat/bitcore lib doge

### PR DESCRIPTION
**Pull request by bsnowden3**

### CODE REVIEW

File: packages/bitcore-lib-doge/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+Contributing to Litecore
```
Comment (matiu) [inline packages/bitcore-lib-doge/CONTRIBUTING.md:1]: rm this file?

File: packages/bitcore-lib-doge/benchmark/script.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+'use strict';
```
Comment (matiu) [inline packages/bitcore-lib-doge/benchmark/script.js:1]: rm this file?

File: packages/bitcore-lib-doge/bower.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (matiu) [inline packages/bitcore-lib-doge/bower.json:1]: bower not used anymore.

File: packages/bitcore-lib-doge/benchmark/script.js (hunk 2 above)
Comment (bsnowden3) [reply packages/bitcore-lib-doge/benchmark/script.js]: removed

**Review by matiu — APPROVED** [review]

**Review by KaycoinsInc — APPROVED** [review]

**Review by KaycoinsInc — APPROVED** [review]

File: packages/bitcore-lib-doge/CONTRIBUTING.md (hunk 1 above)
Comment (themar7777) [reply packages/bitcore-lib-doge/CONTRIBUTING.md]: Run Access

**Pull request merged**